solidity/random-beacon/tasks/staking.ts

```diff
@@ -0,0 +1,123 @@
+import { task, types } from "hardhat/config"
```

Comment:
Should we also rename the file to `stake.ts` to reflect the task name?

Reply:
https://github.com/keep-network/keep-core/pull/3026#discussion_r911824771